### DISCUSSION TITLE
MSSql load error [Jun 18]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2484] [Discord Threads]
> 🙋‍♂️ **Paul**  
#### What should I do ? Thx  
![image0](https://cdn.discordapp.com/attachments/661905455894888493/987004544724652052/unknown.png)

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### <@968449792541884476> - Help us by raising an issue with relevant details here in github (https://github.com/nocodb/nocodb/issues). If you can share a sample redacted data, it would be of great help  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/987004545009872927)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
